outport/factory/outportFactory_test.go

```diff
@@ -104,3 +107,46 @@ func TestCreateOutport_SubscribeNotifierDriver(t *testing.T) {
 
 	require.True(t, outPort.HasDrivers())
 }
+
+func TestCreateOutport_SubscribeMultipleHostDrivers(t *testing.T) {
```

Comment:
we need to add a test with 3 hosts having one inactive. Can be the one with index 1.

Reply:
:+1: